docs/06-concepts/11-authentication/05-custom-overrides.md

```diff
@@ -42,6 +46,36 @@ List<String> scopes = extractScopes(token);
 Set<Scope> userScopes = scopes.map((scope) => Scope(scope)).toSet();
 ```
 
+### Handling revoked authentication 
+
+When a user's authentication is revoked, the server must be promptly notified to maintain secure access control. The `authenticationRevoked` event plays a crucial role in closing method streams that the user is no longer authorized to access. To trigger this event, invoke the `session.messages.authenticationRevoked` method.
+
+```dart
+await session.messages.authenticationRevoked(
+  userId,
+  RevokedAuthenticationScope(
+    scopes: removedScopes,
+  ),
+);
+```
+
+##### Parameters
+
+- `userId` - The user id for the concerned user. This value should match the `userId` field in the `AuthenticationInfo` object.
```

Comment:
If opting for maximum clarity, I would suggest to be more explicit
```suggestion
- `userId` - The user id of the user to be revoked. This value should match the `userId` field in the `AuthenticationInfo` object.
```
Although I would challenge this to be written as this for brevity:
```suggestion
- `userId` - The user id belonging to the `AuthenticationInfo` object to be revoked.
```

Reply:
Great suggestion! I will use the second one; it seems like a good simplification!